lib/annotate_rb/model_annotator/column_annotation/annotation_builder.rb

```diff
@@ -28,6 +29,11 @@ def build
             @column.name
           end
 
+          if @enum_info
```

Comment:
Could we also add an option flag for these changes? This would allow people currently using postgres + enums to opt-in if they want these changes, and to not introduce new unwanted changes.